lib/rubocop/cop/minitest/file_name.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Checks if test file names start with `test_` or end with `_test.rb`.
+      #
+      # @example
+      #   # bad
+      #   my_class.rb
+      #
+      #   # good
+      #   my_class_test.rb
+      #   test_my_class.rb
+      #
+      class FileName < Base
+        include MinitestExplorationHelpers
+
+        MSG = 'Test file path should end with `_test.rb`.'
+
+        def on_class(node)
+          return unless test_class?(node)
+
+          add_offense(node) unless valid_file_name?
```

Comment:
I think, the offense should be added to the whole file, like in `Lint/EmptyFile` cop.

Reply:
Updated to use use global message instead.